app/views/users/_form.html.haml

```diff
@@ -5,6 +5,9 @@
       = f.text_field :spire, placeholder: '12345678@umass.edu'
       = f.form_group do
         = f.check_box :active
-      =f.form_group do
+      = f.form_group do
         = f.check_box :admin
+      = f.text_field :title, help: 'Will be shown on brochure', list: 'title-list'
```

Comment:
```suggestion
      = f.text_field :title, help: 'Will be shown on brochure', list: 'title-list', autocomplete: 'off'
```
I generally prefer turning autocomplete off for `datalist`s, because it'll suggest "bad" data if you made a mistake in the past, but I don't think I'll require this here. Up to you.

Reply:
Funny enough, I was also having an issue where LastPass was offering to fill in "title" with my "title" ("Mr").

Reply:
Of course... Lastpass doesn't believe in `autocomplete='off'`.